bobikApp/TableVC.swift

```diff
@@ -16,43 +16,53 @@ class TableVC: UITableViewController {
         tableView.estimatedRowHeight = 100
     }
     
-    lazy var people: [Person] = {
-           var peops = [Person]()
-           for i in 1...20 {
-               let man = Person(name: "Bob \(i)")
-               peops.append(man)
-           }
-           return peops
-       }()
-
+    lazy var people = {
+        var peops = [Person]()
+        for i in 1...20 {
+            let man = Person(name: "Bob \(i)")
+            peops.append(man)
+        }
+        return peops
+    }()
+    
+    lazy var openedCellStatus: [Bool] = {
+        return Array(repeating: false, count: people.count)
+    }()
+    
```

Comment:
Since only one cell can be expanded at a time, it makes no sense to track the status of the whole array. The index of the expanded cell can be used here.